src/Dfc.CourseDirectory.Database/Pttcd/Views/FindACourseIndex.sql

```diff
@@ -0,0 +1,61 @@
+﻿CREATE VIEW [Pttcd].[FindACourseIndex] WITH SCHEMABINDING
+	AS SELECT
```

Comment:
I'd probably add a description comment that explains that this view is for the search indexer to give future readers an easier clue why this is here than the name.
```suggestion
	AS SELECT
	-- This view is depended on by the Azure Search Indexer which we use to support searching the course data

```